### PR TITLE
test: skip onramp onboarding for returning user

### PR DESCRIPTION
**Pull request by bkirb**
## **Description**

This tests that a returning user in the buy flow _does not_ see the onboarding flow again, or the steps to select region and payment method. It should skip to the build quote screen. 

## **Related issues**

https://consensyssoftware.atlassian.net/browse/RAMPS-1871

## **Manual testing steps**

All Onramp tests are passing

```
 PASS  e2e/specs/ramps/onramp.spec.js (52.479 s)
  smokeAssets Buy Crypto
    ✓ should select Region and Payment Method to see the Build Quote screen (16291 ms)
    ✓ should skip to the Build Quote screen for returning user (14077 ms)

Test Suites: 1 passed, 1 total
Tests:       2 passed, 2 total
Snapshots:   0 total
Time:        52.55 s, estimated 55 s
```

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [x] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: e2e/selectors/Ramps/BuildQuote.selectors.js [hunk 1]
```diff
@@ -3,4 +3,5 @@ import enContent from '../../../locales/languages/en.json';
 export const BuildQuoteSelectors = {
   AMOUNT_TO_BUY_LABEL: enContent.fiat_on_ramp_aggregator.amount_to_buy,
   GET_QUOTES_BUTTON: enContent.fiat_on_ramp_aggregator.get_quotes,
+  CANCEL_BUTTON: enContent.navigation.cancel,
```
Comment (chrisleewilcox) [inline e2e/selectors/Ramps/BuildQuote.selectors.js:6]: Looks like this is a text selector so maybe change to `CANCEL_BUTTON_TEXT`

**Comment by bkirb** [comment]
@chrisleewilcox I updated based on your feedback and tests are still passing ✅
https://github.com/MetaMask/metamask-mobile/pull/11591/commits/2f85d6bfe71fa902780d99ad26937c7a47d517da

```
 PASS  e2e/specs/ramps/onramp.spec.js (54.788 s)
  smokeAssets Buy Crypto
    ✓ should select Region and Payment Method to see the Build Quote screen (17275 ms)
    ✓ should skip to the Build Quote screen for returning user (16008 ms)

Test Suites: 1 passed, 1 total
Tests:       2 passed, 2 total
```

File: e2e/selectors/Ramps/BuildQuote.selectors.js (hunk 1 above)
Comment (bkirb) [reply e2e/selectors/Ramps/BuildQuote.selectors.js]: Added `_TEXT`

**Review by SamuelSalas — DISMISSED** [review]
LGTM

**Review by cortisiko — APPROVED** [review]
✅ ✅ ✅

**Pull request merged**